components/aws/sagemaker/batch_transform/component.yaml

```diff
@@ -0,0 +1,21 @@
+name: 'SageMaker Batch Transformation'
+description: |
+  Batch Transformation Jobs in SageMaker
+metadata:
+  labels: {platform: 'SageMaker'}
+inputs:
+  - {name: region,          description: 'The region where the cluster launches.'}
+  - {name: model_name,      description: 'The name of the model that you want to use for the transform job.'}
+  - {name: input_location,  description: 'The S3 location of the data source that is associated with a channel.'}
+  - {name: output_location, description: 'The Amazon S3 path where you want Amazon SageMaker to store the results of the transform job.'}
+implementation:
+  container:
+    image: seedjeffwan/kubeflow-pipeline-aws-sm:20190501-05
```

Comment:
Nit: can we move to more neutral location for these images, instead of an individual if possible?

Reply:
Good point. Will do.

Reply:
Makes sense